models/customers-search-list-my-view/src/generator.ts

```diff
@@ -0,0 +1,57 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { Generator, fake } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TCustomersSearchListMyView } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+// ref https://github.com/commercetools/merchant-center-services/blob/main/packages/settings/lib/schemas/customers-search-list-my-view.graphql
+const generator = Generator<TCustomersSearchListMyView>({
+  fields: {
+    /**
+     * id: ID!
+     */
+    id: fake((f) => f.string.uuid()),
+    /**
+     * createdAt: DateTime!
+     */
+    createdAt: fake(getOlderDate),
+    /**
+     * updatedAt: DateTime!
+     */
+    updatedAt: fake(getNewerDate),
+    /**
+     * userId: String!
+     */
+    userId: fake((f) => f.string.uuid()),
+    /**
+     * projectKey: String!
+     */
+    projectKey: fake((f) => f.lorem.slug(2)),
+    /**
+     * nameAllLocales [LocalizedField!]
+     */
+    nameAllLocales: null,
+    name: fake(() => LocalizedString.random()),
```

Comment:
I'm sorry but I don't fully understand this property because this model only exists in the GraphQL representation and it does not have this property.
Could you maybe explain why we need it? 🙏 

![image](https://github.com/user-attachments/assets/08e6fcca-25ca-4f5f-b368-dd1d672dd853)

Reply:
Good point! I'm not sure why this field was implemented in this [model here](https://github.com/commercetools/merchant-center-frontend/blob/main/packages-shared/test-data/src/customers-search-list-my-view/generator.js). I'll take a look at the usage and see if i find something. Otherwise I'll update the generator as per graphql

Reply:
Updated https://github.com/commercetools/test-data/pull/654/commits/621ac54137f978068e064114a6fb6754b3482b38